src/event.ts

```diff
@@ -9,6 +9,7 @@ import type {
   ScriptReference,
   PackageReference,
 } from './script.js';
+import {AstNode, ParseError} from './util/ast.js';
```

Comment:
I've been exclusively using `import type` for types. I like how it visually separates actual runtime module dependencies from type-system-only ones.

I think that's the only benefit that applies to us, though. If we wanted to (partly) enforce it, we could set `"importsNotUsedAsValues": "error"`.

But maybe this style is too inconvenient to follow, WDYT? Maybe would be nice if VSCode did this automatically.